.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would prefer not to have this checked in - is it not possible to configure vscode in your home directory or something?